src/coreComponents/linearAlgebra/interfaces/hypre/HypreMGR.cpp

```diff
@@ -74,7 +74,7 @@ void hypre::mgr::createMGR( LinearSolverParameters const & params,
   {
     GEOS_LOG_RANK_0( GEOS_FMT( "        MGR preconditioner: numComponentsPerField = {}", numComponentsPerField ) );
   }
-  if( params.logLevel >= 1024 )
+  if( params.logLevel >= 1 )
```

Comment:
```suggestion
  if( params.logLevel >= 1024 )
```
pointmarkers array is supposed to be printed out only for high enough values of logLevel.

Reply:
we may want to have a dedicated hypre Logging logic.

Reply:
That might be a good idea, 1:1 with hypre's `print_level` to avoid confusions. One the other hand, we would need yet another variable in the code...

Reply:
@arng40 Please revert to the original value as changing loglevel is not the scope of this PR.